src/sql/workbench/contrib/profiler/browser/profilerActions.contribution.ts

```diff
@@ -93,9 +94,23 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 			} else {
 				// clear data when profiler is started
 				profilerInput.data.clear();
-				return profilerService.startSession(profilerInput.id, profilerInput.sessionName);
+				return profilerService.startSession(profilerInput.id, profilerInput.sessionName, false);
 			}
 		}
 		return Promise.resolve(false);
 	}
 });
+
+CommandsRegistry.registerCommand({
+	id: 'profiler.openXELFile',
```

Comment:
The profiler contribution seems to be a general extensibility thing for any extension to use, so we should avoid using the term "XEL" anywhere in the core code. Just generic "file" or something should be fine.